app/transformers/business/application.js

```diff
@@ -0,0 +1,29 @@
+export function transformOrganisationApplicationToBusinessApplications (applications) {
+  if (!applications) {
+    return applications
+  }
+
+  const result = []
+  for (const response in applications) {
+    result.push({
+      applicationStatus: {
+        id: response.application_id,
+        open: response.application_code,
```

Comment:
Doesn't seem right a code in this field will not make sense

Reply:
Could you be more specific or add mapping here so I can see the difference, ticket had no context; I mapped this from spreadsheet.